### PR TITLE
Add basic support for music

### PR DESCRIPTION
**Pull request by anaximander**
It's missing the code for generating the music data, which I can (and probably should) add, since the next "scraper.js" run won't have the music images in imgMap.js.  That code just needs some cleanup since I had trouble scraping from the wiki directly and found an alternative source.  Besides that, does this functionality look OK?

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -484,12 +508,14 @@ function Row({
...
 
     & svg {
+      flex: 1;
     }
   `;
 
   const textStyle = css`
     margin-left: 6px;
     text-align: left;
+    flex: 10;
```
Comment (SawyerHood) [inline src/App.tsx:518]: Is there a reason for this to be 10?

File: src/AppState.ts [hunk 2]
```diff
@@ -101,9 +107,11 @@ function cleanCollectable(input: { [key: string]: any }): ICollectable {
   const key = name
     .toLowerCase()
     .replace(/ /g, "_")
-    .replace("'", "")
-    .replace("-", "_")
-    .replace(".", "");
+    .replaceAll("'", "")
```
Comment (SawyerHood) [inline src/AppState.ts]: We can't use `replaceAll`, this is what is crashing the site. You either need to add a polyfill or use replace with regex and a global flag (see above). We have enough cases we should probably just merge them into a regex

**Review by SawyerHood — CHANGES_REQUESTED** [review]
ReplaceAll is crashing the site, make sure that you fix that. I left a few comments. After that it shoul dbe good to merge. https://caniuse.com/#search=replaceAll

**Comment by SawyerHood** [comment]
It would be rad if you could add documentation on where you got the data from so that we can update in the future if needed

File: src/App.tsx (hunk 1 above)
Comment (anaximander) [reply src/App.tsx:518]: Mostly just that if I didn't specify the flex values for this (and the other child of the flex container: the svg), the SVG would shrink when there was a lot of text.  10:1 seems to preserve the original spacing/sizing (as seen in the "leaving this month" warning).

File: src/AppState.ts (hunk 2 above)
Comment (anaximander) [reply src/AppState.ts]: Ah, good catch.  I must've just gotten "lucky" that I was on Firefox 😂  I'll just swap this out with a regex for global replace in a cross-browser compliant way, my bad.

**Comment by anaximander** [comment]
I'll add the data source/script for sure, just need to clean up that code to get it ready for PR, and make the requested changes to the application code as well.

File: src/AppState.ts (hunk 2 above)
Comment (anaximander) [reply src/AppState.ts]: Should be fixed now (using global regex replace)

**Comment by anaximander** [comment]
Updated scraper (with notes and source csv).  One caveat that I realized while updating the scraper is that the owners of the data source (see comments) ask politely to be credited somehow on the site.  Up to you how to handle that part 🤷

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,6 @@
 node_modules
 build
 .DS_Store
+scraper/*.json
```
Comment (SawyerHood) [inline .gitignore:4]: Thanks for this

**Review by SawyerHood — APPROVED** [review]
Awesome thanks for doing this!!! I'll add some follow ups to make sure that we wrap the tabs on mobile (it is starting to get cramped)

**Pull request merged**